### ISSUE TITLE
[Bug] generate golang code fron testcatse, code is error

### ISSUE THREAD

**Issue by yuluo-yx** [LinuxSuRen/api-testing#442]
1. test case 

![image](https://github.com/LinuxSuRen/api-testing/assets/77964041/3e88024a-3f29-4612-8625-7134233d20b0)

2.  code

```
package main

import (
	"io"
	"net/http"
)

func main() {
	body := bytes.NewBufferString("")

	req, err := http.NewRequest("GET," "http://127.0.0.1:81/check", body)
	if err != nil {
		panic(err)
	}

	resp, err := http.DefaultClient.Do(req)
	if err != nil {
		panic(err)
	}

	if resp.StatusCode != http.StatusOK {
		panic("status code is not 200")
	}

	data, err := io.ReadAll(resp.Body)
	println(string(data))
}
```

3. error

```
NewRequest("GET," "htt
```

```
import (
	"io"
	"net/http"
)

func main() {
	body := bytes.NewBufferString("")
```
not import `bytes` pkg.

**Comment by LinuxSuRen** [comment]
I found some issues related to the special characters.

![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/24064b28-0f37-4a2f-a707-7e89b6c43e97)
![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/89ce62f6-43dc-4556-a238-b0303719c7cb)

I guess we can fix it by changing from `text/template` to `html/template`

https://github.com/LinuxSuRen/api-testing/blob/b40eea5afaca9e873ece02e54df652d802af1cda/pkg/generator/golang_generator.go#L20

**Comment by yuluo-yx** [comment]
> I found some issues related to the special characters.
> 
> ![image](https://private-user-images.githubusercontent.com/1450685/331915539-24064b28-0f37-4a2f-a707-7e89b6c43e97.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYxOTAwOTcsIm5iZiI6MTcxNjE4OTc5NywicGF0aCI6Ii8xNDUwNjg1LzMzMTkxNTUzOS0yNDA2NGIyOC0wZjM3LTRhMmYtYTcwNy03ZTg5YjZjNDNlOTcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjBUMDcyMzE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YmZmMjVmMWE2MWZlYjNkMDc0ZGVhNTM3ZGJmOTA5ZTA4ZTRkNzk0YmViOTE4YjljMDEwZDliNzAzNDE4MTkyYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.ObgRxVTKAb3MoD_Jp7HhEWyYjJKuMyoAB8_7NY03ioY) ![image](https://private-user-images.githubusercontent.com/1450685/331915571-89ce62f6-43dc-4556-a238-b0303719c7cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYxOTAwOTcsIm5iZiI6MTcxNjE4OTc5NywicGF0aCI6Ii8xNDUwNjg1LzMzMTkxNTU3MS04OWNlNjJmNi00M2RjLTQ1NTYtYTIzOC1iMDMwMzcxOWM3Y2IucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjBUMDcyMzE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjlhZTgzM2YxNzk1ZmVlYWQxOWJhYzFiZDlkMWVhZWY4MWQ2NzJiMGUyMWM4MGNkMTIwYjBmYjQ0MWRjOGM2OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.8mZ0MtWJ_ezmjr19qVGU_m2_C9bG8aZk7P2n1ms5KPU)
> 
> I guess we can fix it by changing from `text/template` to `html/template`
> 
> https://github.com/LinuxSuRen/api-testing/blob/b40eea5afaca9e873ece02e54df652d802af1cda/pkg/generator/golang_generator.go#L20

cc @wt-goodluck

**Comment by wt-goodluck** [comment]
ok,I'll take a look at it tonight

**Comment by LinuxSuRen** [comment]
fixed by #457
